digdag-ui/model.js

```diff
@@ -70,7 +70,7 @@ export type Task = {
 export type Attempt = {
   id: number;
   project: IdName;
-  workflow: NameOptionalId;
+  workflow: NameId;
```

Comment:
The `Attempt` type should probably match the underlying rest api type.

https://github.com/treasure-data/digdag/blob/master/digdag-client/src/main/java/io/digdag/client/api/RestSessionAttempt.java#L23